admin/src/config.js

```diff
@@ -1,17 +1,27 @@
 const location = window.location;
 
 function buildPrefixedApiUri(prefix) {
+  let parts = location.hostname.split('.');
```

Comment:
🤢

Reply:
yeah.. maybe let's just hardcode for now till we figure this out in #175?